app/model/exactTarget/DataExtensionRow.scala

```diff
@@ -339,8 +338,8 @@ object GuardianWeeklyRenewalDataExtensionRow {
       title = contact.title.flatMap(Title.fromString),
       firstName = contact.firstName.getOrElse(""),
       lastName = contact.lastName,
-      startDate = contractEffective,
-      firstPaymentDate = customerAcceptance,
+      startDate = newTermStartDate,
```

Comment:
The fulfilment (re)start date and renewal first payment date are always the same date for renewals.